docker-compose.yml

```diff
@@ -1,31 +1,13 @@
 version: "3.8"
 
 services:
-  mysql:
+  db:
     image: mysql:8.0.23
-    environment:
-      MYSQL_DATABASE: 'yourdbname'
-      MYSQL_USER: 'user'
-      MYSQL_PASSWORD: 'password'
-      MYSQL_ROOT_PASSWORD: 'rootpassword'
-    ports:
-      - "3306:3306"
-    volumes:
-      - mysql-data:/var/lib/mysql
     restart: always
-
-  app:
-    image: your-spring-boot-app:latest
-    depends_on:
-      - mysql
     environment:
-      SPRING_DATASOURCE_URL: jdbc:mysql://mysql:3306/yourdbname?useSSL=false&serverTimezone=UTC
-      SPRING_DATASOURCE_USERNAME: user
-      SPRING_DATASOURCE_PASSWORD: password
+      MYSQL_DATABASE: puresynth
+      MYSQL_USER: ${DB_USERNAME}
+      MYSQL_PASSWORD: ${DB_PASSWORD}
+      MYSQL_ALLOW_EMPTY_PASSWORD: "yes"
     ports:
-      - "8080:8080"
-    restart: always
-
-volumes:
-  mysql-data:
-    driver: local
\ No newline at end of file
+      - "3307:3306"
```

Comment:
You should configure the IDE to put EOF on save: https://stackoverflow.com/questions/16761227/how-to-make-intellij-idea-insert-a-new-line-at-every-end-of-file